setup/module/api/variables.tf

```diff
@@ -0,0 +1,55 @@
+# todo: descriptions
+
+variable "name" {}
+
+variable "region" {}
+
+variable "version" {}
+
+variable "vpc_id" {}
+
+variable "username" {}
+
+variable "password" {}
+
+variable "ssh_key_pair" {}
+
+variable "dockercfg" {}
+
+variable "tags" {
+  description = "A map of tags to add to all resources"
+  default     = {}
+}
+
+# Current AMI: amzn-ami-2016.03.c-amazon-ecs-optimized
```

Comment:
as long as we're releasing a new major version, we might as well bump up to a newer AMI